test/libsolidity/SemanticTest.cpp

```diff
@@ -74,6 +74,7 @@ bool SemanticTest::run(ostream& _stream, string const& _linePrefix, bool _format
 
 		test.setFailure(!m_transactionSuccessful);
 		test.setRawBytes(std::move(output));
+		test.setContractABI(m_compiler.contractABI(m_compiler.lastContractName()));
```

Comment:
What about tests with multiple contracts?

Reply:
Right now, you can't call functions on a contract that is not the recently compiled one directly, which means that only the last compiled ABI is of interest.